### PR TITLE
redmine commands - handle multiselect customfield

### PR DESCRIPTION
**Pull request by tcarmeli1**
## Status
- [x] Ready

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-37800

## Description
handle multiselect in custom field argument

### CODE REVIEW

File: Packs/Redmine/ReleaseNotes/1_0_4.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Redmine
+
+- Updated the custom_fields argument in the **redmine-issue-create**, **redmine-issue-update** to handle multiselect fields.
```
Comment (jbabazadeh) [inline Packs/Redmine/ReleaseNotes/1_0_4.md:6]:
```suggestion
- Fixed an issue where custom_fields argument in the **redmine-issue-create**, **redmine-issue-update** didn't handle multiselect fields.
```

File: Packs/Redmine/Integrations/Redmine/Redmine.py [hunk 2]
```diff
@@ -165,15 +165,17 @@ def adjust_paging_to_request(page_number, page_size, limit):
...
+            for key, value in custom_fields_dict.items():
+                custom_fields_for_request.append({"id": key, "value": value})
+            args['custom_fields'] = custom_fields_for_request
         except Exception as e:
-            if 'list index out of range' in e.args[0] or 'substring not found' in e.args[0]:
-                raise DemistoException("Custom fields not in format, please follow the instructions")
-            raise
+            raise DemistoException(
+                f"Custom fields not in format, please follow this format:"
+                f" `{{\"customFieldID1\": [\"value1\",\"value2\"], \"customFieldID2\": \"value3\"}}`. with error: {e}")
```
Comment (jbabazadeh) [inline Packs/Redmine/Integrations/Redmine/Redmine.py:178]: change the order of the customFields.. (the common option should be first) and add note if it multiselect to add  as a list.

File: Packs/Redmine/Integrations/Redmine/Redmine.yml [hunk 3]
```diff
@@ -66,7 +66,7 @@ script:
     - name: parent_issue_id
       description: The ID of the parent issue.
     - name: custom_fields
-      description: The custom field to update. The format is costumFieldID:Value,costumFieldID:Value, etc.
+      description: "The custom field to update. The format is `{\"customFieldID1\": [\"value1\",\"value2\"], \"customFieldID2\": \"value3\"}` - use array if the field is of multiselect type."
```
Comment (jbabazadeh) [inline Packs/Redmine/Integrations/Redmine/Redmine.yml:69]: same here

**Review by jbabazadeh — APPROVED** [review]
Great job!

File: Packs/Redmine/Integrations/Redmine/README.md [hunk 4]
```diff
@@ -53,7 +53,7 @@ After you successfully execute a command, a DBot message appears in the War Room
 | fixed_version_id | The target version ID for this issue. | Optional | 
 | assigned_to_id | The ID of the user to assign the issue to. | Optional | 
 | parent_issue_id | The ID of the parent issue. | Optional | 
-| custom_fields | The custom field to update. The format is costumFieldID:Value,costumFieldID:Value, etc. | Optional | 
+| custom_fields | - The custom field to update.<br/>- "The format is `{\"customFieldID1\": [\"value1\",\"value2\"], \"customFieldID2\": \"value3\"}`."<br/>- Use an array if the field is of multiselect type.<br/>- Instruction for each custom field type:<br/>  - Boolean: use "0" for False and "1" for True.<br/>  - Date: the format is "yyyy-mm-dd".<br/>  - Float: A number with two digits after the decimal point.<br/>  - Integer: A number.<br/>  - Key/Value list: Use the Key (ID) of the value. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Redmine/Integrations/Redmine/README.md:56]:
```suggestion
| custom_fields | The custom field to update.<br/>- "The format is `{\"customFieldID1\": [\"value1\",\"value2\"], \"customFieldID2\": \"value3\"}`."<br/>- Use an array if the field is of multiselect type.<br/>- Instruction for each custom field type:<br/>  - Boolean: use "0" for False and "1" for True.<br/>  - Date: the format is "yyyy-mm-dd".<br/>  - Float: A number with two digits after the decimal point.<br/>  - Integer: A number.<br/>  - Key/Value list: Use the Key (ID) of the value. | Optional | 
```

File: Packs/Redmine/Integrations/Redmine/Redmine.yml [hunk 5]
```diff
@@ -66,7 +66,16 @@ script:
     - name: parent_issue_id
       description: The ID of the parent issue.
     - name: custom_fields
-      description: The custom field to update. The format is costumFieldID:Value,costumFieldID:Value, etc.
+      description: |-
+        - The custom field to update.
```
Comment (ShirleyDenkberg) [inline Packs/Redmine/Integrations/Redmine/Redmine.yml:70]:
```suggestion
        The custom field to update.
```

File: Packs/Redmine/Integrations/Redmine/Redmine.yml [hunk 6]
```diff
@@ -202,7 +211,16 @@ script:
     - name: parent_issue_id
       description: The ID of the parent issue, if applicable.
     - name: custom_fields
-      description: The custom field to update. The format is costumFieldID:Value,costumFieldID:Value etc. 
+      description: |-
+        - The custom field to update.
```
Comment (ShirleyDenkberg) [inline Packs/Redmine/Integrations/Redmine/Redmine.yml:215]:
```suggestion
        The custom field to update.
```

File: Packs/Redmine/ReleaseNotes/1_0_4.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Redmine
+
+- Fixed an issue where custom_fields argument in the **redmine-issue-create**, **redmine-issue-update** didn't handle multiselect fields.
```
Comment (ShirleyDenkberg) [inline Packs/Redmine/ReleaseNotes/1_0_4.md:6]:
```suggestion
Fixed an issue where the *custom_fields* argument in the ***redmine-issue-create*** and ***redmine-issue-update*** commands didn't handle multiselect fields.
```

**Comment by ShirleyDenkberg** [comment]
@jbabazadeh Doc review completed.

**Pull request merged**